server/lib/query-parser.js

```diff
@@ -0,0 +1,6 @@
+import url from 'url'
+
+export default function queryParser (req, res, next) {
+  req.query = url.parse(req.url, true).query
```

Comment:
As per https://github.com/zeit/next.js/pull/4979/files#r210998217

This introduces a non-standard side-effect in development and it's not really needed

Reply:
Solved this by adding a custom middleware: https://github.com/zeit/next.js/pull/4979/commits/b1663359744f5bf6843e12a79956602f64d97fef#diff-b426d86a05ba55110073f5db11a1068c